test/quantization/core/test_quantized_op.py

```diff
@@ -4624,7 +4624,7 @@ def _test_qconv_impl(
             assert not use_transpose, "Cannot fuse ReLU with ConvTranspose"
             relu = torch.nn.ReLU()
             result_ref = relu(result_ref)
-        elif post_op == 'add':
+        elif post_op == 'add' or post_op == 'add_relu':
```

Comment:
nit: might be clearer to duplicate this code and put "add_relu" in the top level branch
```
elif post_op == “add":
   ...
elif post_op == "add_relu":
   ...
```

Reply:
Sure, Put the "add_relu" in the top level branch to make the code clearer.